src/Components/login/Login.js

```diff
@@ -20,8 +20,15 @@ const Login = () => {
   const [showPassword, setShowPassword] = useState(false);
   const [isLoggedIn, setIsLoggedIn] = useState(false);
   const [goCreateAccount, setGoCreateAccount] = useState(false);
+  const informationUser = JSON.parse(localStorage.getItem("userInformation"));
   const { classes } = styles();
 
+  useEffect(() => {
+    if (informationUser.isLoggedIn) {
```

Comment:
keep in mind that at any point `informationUser` can be null, so this will break:
` if (informationUser && informationUser.isLoggedIn) {`